mantle/platform/api/esx/api.go

```diff
@@ -444,12 +444,6 @@ func (a *API) deleteDevice(vm *object.VirtualMachine) error {
 }
 
 func (a *API) buildCreateImportSpecRequest(name string, ovaPath string, finder *find.Finder, defaultNetwork object.NetworkReference, resourcePool *object.ResourcePool, datastore *object.Datastore) (*archive, *types.OvfCreateImportSpecResult, error) {
-	var nets []types.OvfNetworkMapping
-	nets = append(nets, types.OvfNetworkMapping{
-		Name:    "mantle",
-		Network: defaultNetwork.Reference(),
```

Comment:
Hmm, could `defaultNetwork.Reference()` have some side effect we're relying on?  It seems like no.

In this type of situation I would probably do some investigation with `git log` to see *why* this code is dead.  It might actually be a different bug.

I did this in e.g. https://github.com/coreos/ignition/pull/1133 where it turned out the code was no longer used (intentionally) as opposed to because of a separate bug.

In this case you'd have to dig back into the original mantle git history.

Reply:
(On the other hand, I don't know that we're using this code, so we could just delete it)

Reply:
Thank you for review. That's a good habit. I will keep in mind and check the logs.

Reply:
> (On the other hand, I don't know that we're using this code, so we could just delete it)

I'm not sure the ESX code has ever really been used by anything ever since it was initially written. Originally we were going to have some hardware to run these on in CL but it never manifested.

Reply:
> Hmm, could defaultNetwork.Reference() have some side effect we're relying on? It seems like no.

Quick glance through `vmware/govmomi` doesn't make it seem like there's any adverse side effects, it's just a getter for a managed reference object.

Additionally digging through it looks like this was dead code at the time it was introduced (likely do to review refactoring) and was never caught.

Should be safe to just drop it.

Reply:
Thank you for confirming.